internal/services/containers/kubernetes_cluster_resource.go

```diff
@@ -160,6 +160,17 @@ func resourceKubernetesCluster() *pluginsdk.Resource {
 				}, false),
 			},
 
+			"node_os_channel_upgrade": {
+				Type:     pluginsdk.TypeString,
+				Optional: true,
+				ValidateFunc: validation.StringInSlice([]string{
+					string(managedclusters.NodeOSUpgradeChannelNodeImage),
+					string(managedclusters.NodeOSUpgradeChannelSecurityPatch),
+					string(managedclusters.NodeOSUpgradeChannelUnmanaged),
+					string(managedclusters.NodeOSUpgradeChannelNone),
+				}, false),
+			},
```

Comment:
The properties in the schema are ordered alphabetically, so this will need to be moved down.

Reply:
done.